code/modules/mob/mob_helpers.dm

```diff
@@ -445,8 +445,9 @@ It's fairly easy to fix if dealing with single letters but not so much with comp
 
 /proc/offer_control(mob/M)
 	M << "Control of your mob has been offered to dead players."
-	log_admin("[key_name(usr)] has offered control of ([key_name(M)]) to ghosts.")
-	message_admins("[key_name_admin(usr)] has offered control of ([key_name_admin(M)]) to ghosts")
+	if(usr)
+		log_admin("[key_name(usr)] has offered control of ([key_name(M)]) to ghosts.")

```

Comment:
Yes, during early development. I threw this sanity check in to make sure it wouldn't happen again.